ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/TopAppBar.kt

```diff
@@ -285,15 +349,64 @@ private fun TopBarOneVisibleActionMenuExample(
     )
 }
 
+@Composable
+fun TopAppBarBackNavigation(
+    backContentDescriptionText: String? = null,
+    backIconVector: ImageVector = Icons.AutoMirrored.Filled.ArrowBack,
+    backText: String? = null,
+    onBack: () -> Unit
+) {
+    Box(
```

Comment:
u can remove the Box and use Row only.  The following should do the trick (modifier order matters)
`Modifier
                .wrapContentSize()
                .clip(RoundedCornerShape(ZcashTheme.dimens.regularRippleEffectCorner))
                .clickable { onBack() }
                .padding(all = ZcashTheme.dimens.spacingDefault)`

Reply:
Yes, that's a good catch. That is probably leftover from the previous implementation. I also changed the Row padding to align with the size of the other TopAppBar buttons.